yarn.lock

```diff
@@ -30219,6 +30229,14 @@ url-parse@^1.4.3:
     querystringify "^2.1.1"
     requires-port "^1.0.0"
 
+url-parse@^1.4.4:
```

Comment:
It seems like you've added these libraries? If you're going to add libraries, we should consider adding ones that conform to the URL api